src/main/kotlin/apply/domain/administrator/Administrator.kt

```diff
@@ -10,15 +10,16 @@ import javax.persistence.Entity
 @Entity
 class Administrator(
     @Column(unique = true)
-    private val name: String,
+    val name: String,
+    private val username: String,
```

Comment:
r: 로그인에 사용되는 `username`은 고유해야 합니다.

```suggestion
    private val username: String,
    val name: String,
```